src/analyses/goto_check.cpp

```diff
@@ -342,9 +342,9 @@ void goto_checkt::enum_range_check(const exprt &expr, const guardt &guard)
   const c_enum_typet::memberst enum_values = c_enum_type.members();
 
   std::vector<exprt> disjuncts;
-  for(auto it = enum_values.begin(); it != enum_values.end(); ++it)
+  for(const auto it : enum_values)
```

Comment:
`const auto &`, and `it` -> `enum_value` since it's not longer an iterator

Reply:
Done, thanks